WordPress/src/main/res/layout/login_2fa_screen.xml

```diff
@@ -64,10 +64,12 @@
                     style="@style/Base.TextAppearance.AppCompat.Caption"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:minHeight="48dp"
```

Comment:
Just noticed there's no visual feedback when you tap this link. Adding `android:background="?android:selectableItemBackground"` would resolve that.